src/lib/es2020.bigint.d.ts

```diff
@@ -384,14 +384,19 @@ interface BigInt64ArrayConstructor {
      */
     of(...items: bigint[]): BigInt64Array;
 
+    /**
+     * Creates an array from an array-like or iterable object.
+     * @param arrayLike An array-like or iterable object to convert to an array.
+     */
+    from(arrayLike: ArrayLike<bigint> | Iterable<bigint>): BigInt64Array;

```

Comment:
why do these differ from the types in TypeArray.from? For that one, we just use `Iterable`.

Some guesses:
- es2020.bigint can be used without including es2015.iterable (or whatever file makes Arrays Iterable)
- Iterable is added here because it's in TypedArray.from, but ArrayLike stays for backward compatibility.

Reply:
Because in those cases, the `ArrayLike` overloads had already been defined in es5 prior to the introduction of iterables, so the `ArrayLike` signature and the `Iterable` signature exist as two separate overloads (_eg._ [here](https://github.com/microsoft/TypeScript/blob/v5.5-rc/src/lib/es5.d.ts#L2119-L2123) and [here](https://github.com/microsoft/TypeScript/blob/v5.5-rc/src/lib/es5.d.ts#L2119-L2123)). Since the bigint typed arrays are post-es2015, it's possible to have these as one single overload.

`ArrayLike` objects don't have to be iterable, so the method definitions need to accept both:
```ts
BigInt64Array.from(
  // ArrayLike<number> but not iterable
  {
    0: 12345678,
    1: 98765432
    length: 2
  },
  (n: number): bigint => BigInt(n) ** 2n,
) // BigInt64Array [ 152415765279684n, 9754610558146624n ]
```

Reply:
On that subject, though, the bigint TypedArray constructors in es2020.bigint don't look like they have a `new (ArrayLike<bigint>)` signature, which probably also needs addressing.